<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Точка в пространстве</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Аксонометрическая система плоскостей</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="aWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>250</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Комплексный чертеж</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="cWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>250</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QGroupBox" name="coordinatesGroupBox">
          <property name="title">
           <string>Координаты точки T</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="2">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLineEdit" name="xField">
               <property name="minimumSize">
                <size>
                 <width>35</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>35</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="yField">
               <property name="minimumSize">
                <size>
                 <width>35</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>35</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="zField">
               <property name="minimumSize">
                <size>
                 <width>35</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>35</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>X:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Y:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Z:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>-100</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>100</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QSlider" name="xSlider">
                 <property name="minimumSize">
                  <size>
                   <width>250</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="minimum">
                  <number>-100</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="invertedAppearance">
                  <bool>false</bool>
                 </property>
                 <property name="invertedControls">
                  <bool>false</bool>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksAbove</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>-100</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>100</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QSlider" name="ySlider">
                 <property name="minimum">
                  <number>-100</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="invertedAppearance">
                  <bool>false</bool>
                 </property>
                 <property name="invertedControls">
                  <bool>false</bool>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksAbove</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>-100</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>100</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QSlider" name="zSlider">
                 <property name="minimum">
                  <number>-100</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="invertedAppearance">
                  <bool>false</bool>
                 </property>
                 <property name="invertedControls">
                  <bool>false</bool>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksAbove</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QGroupBox" name="angleComboBox">
          <property name="title">
           <string>Угол поворота</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>X:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Y:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Z:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_15">
                   <property name="text">
                    <string> 0°</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_16">
                   <property name="text">
                    <string>360°</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QSlider" name="xAngleSlider">
                 <property name="minimumSize">
                  <size>
                   <width>250</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>360</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="invertedAppearance">
                  <bool>false</bool>
                 </property>
                 <property name="invertedControls">
                  <bool>false</bool>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksAbove</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_17">
                   <property name="text">
                    <string> 0°</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_6">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_18">
                   <property name="text">
                    <string>360°</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QSlider" name="yAngleSlider">
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>360</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="invertedAppearance">
                  <bool>false</bool>
                 </property>
                 <property name="invertedControls">
                  <bool>false</bool>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksAbove</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <item>
                  <widget class="QLabel" name="label_19">
                   <property name="text">
                    <string> 0°</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_7">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_20">
                   <property name="text">
                    <string>360°</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QSlider" name="zAngleSlider">
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>360</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="invertedAppearance">
                  <bool>false</bool>
                 </property>
                 <property name="invertedControls">
                  <bool>false</bool>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksAbove</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="0" column="2">
            <layout class="QVBoxLayout" name="verticalLayout_14">
             <item>
              <widget class="QLineEdit" name="xAngleField">
               <property name="minimumSize">
                <size>
                 <width>35</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>35</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="yAngleField">
               <property name="minimumSize">
                <size>
                 <width>35</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>35</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="zAngleField">
               <property name="minimumSize">
                <size>
                 <width>35</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>35</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>xSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_x_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_y_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_z_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xAngleSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_x_angle_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>563</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zAngleSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_z_angle_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>563</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yAngleSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_y_angle_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>563</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_x_changed(int)</slot>
  <slot>on_y_changed(int)</slot>
  <slot>on_z_changed(int)</slot>
  <slot>on_x_angle_changed(int)</slot>
  <slot>on_y_angle_changed(int)</slot>
  <slot>on_z_angle_changed(int)</slot>
 </slots>
</ui>
